constraint/pkg/client/client.go

```diff
@@ -588,6 +595,13 @@ func (c *Client) review(ctx context.Context, target string, constraints []*unstr
 		Trace:   trace,
 		Target:  target,
 		Results: results,
+		ResponseMeta: &types.ResponseMeta{
```

Comment:
The intended use case for this feature would be: if I have 5 constraint templates and one of them is slow, I would like to identify which template is the culprit.

In order to do that, I'd need to be able to associate a given constraint template with its total runtime.

TL;DR we need to break down these `Meta` objects by constraint template name... probably by modifying the engine interface to allow that.